app/public/css/styles.css

```diff
@@ -764,6 +775,13 @@ button[data-balloon] {
   white-space: pre-wrap;
 }
 
+.modal-instructions {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-right: 24px;
+  margin-left: 24px;
+}
```

Comment:
this can be shortened to 
.modal-instructions {
  margin: 20px 24px;
}

read more about shorthand syntax here: https://www.w3schools.com/css/css_margin.asp

Reply:
fixed